Firmware/firmwareLiBCM/config.h

```diff
@@ -95,7 +95,7 @@
 
     #define CELL_VMAX_REGEN                     43000 //43000 = 4.3000 volts
     #define CELL_VMIN_ASSIST                    31900
-    #define CELL_VMAX_GRIDCHARGER               39600 //3.9 volts is 75% SoC //other values: See SoC.cpp //MUST be less than 'CELL_VREST_85_PERCENT_SoC'
+    #define CELL_VMAX_GRIDCHARGER               39800 //3.9 volts is 75% SoC //other values: See SoC.cpp //MUST be less than 'CELL_VREST_85_PERCENT_SoC'
```

Comment:
Increasing the grid charger Vcell limit from 3.96 V/cell to 3.98 V/cell is problematic for people who live on top of steep mountains.  Your new value (~84% SoC) butts right up against LiBCM's 85% "disable regen" limit, which means hill people won't be able to regen downhill after grid charging.

Reply:
oh sorry this was accidental!